AdyenTwint/AdyenTwint.h

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for AdyenTwint.
+FOUNDATION_EXPORT double AdyenTwintVersionNumber;
```

Comment:
Hmm, why do we need these values for Twint?

Reply:
These files came automatically when I created the new target for Twint.